open-problems.md

```diff
@@ -0,0 +1,5 @@
+# Open problems
```

Comment:
I would rename this to `Open Research Problems`

Reply:
I suggest we do this rename in https://github.com/vacp2p/vac.dev/issues/81

Reply:
fixed, this belongs in a different PR though